### ISSUE TITLE
PHPUnit Test failures are not properly notified when going from 0.9.0 to 0.9.1

### ISSUE THREAD

**Issue by patternpaul** [mikeerickson/gulp-phpunit#27]
It seems that since I've upgraded to gulp-phpunit 0.9.1 from 0.9.0, on error notifications fail to fire. Here are some screenshots of the packages, cases and gulp file.
0.9.0
![image](https://cloud.githubusercontent.com/assets/3682501/9648642/47472144-51b0-11e5-8f85-a44965fd2a24.png)
![image](https://cloud.githubusercontent.com/assets/3682501/9648654/68a16ffc-51b0-11e5-9bb0-49487bcef546.png)
0.9.1
![image](https://cloud.githubusercontent.com/assets/3682501/9648672/851e9b5a-51b0-11e5-88c6-27852490591e.png)
![image](https://cloud.githubusercontent.com/assets/3682501/9648680/a006c4ce-51b0-11e5-9640-e47e63871636.png)

Here is my gulp file
```javascript

var gulp = require('gulp');
var run = require('gulp-run');
var notify = require('gulp-notify');
var phpunit = require('gulp-phpunit');


gulp.task('phpunit', function() {
    gulp.src('phpunit.xml')
        .pipe(phpunit('', {debug: false, notify: true}))
        .on('error', notify.onError({
            title: 'PHPUnit Failed',
            message: 'One or more tests failed.'
        }))
        .pipe(notify({
            title: 'PHPUnit Passed',
            message: 'All tests passed!'
        }));
});


gulp.task('watch', function() {
    gulp.watch(['tests/**/*.php', 'App/**/*.php'], ['phpunit']);
});

```

**Comment by sstok** [comment]
Same problem here.

**Comment by mikeerickson** [comment]
Fixed in 0.10.x release
